query.py

```diff
@@ -167,41 +183,27 @@ def query_node(node, http, url, on_query_error=None):
     return node.bind(dres)
 
 
+=======
+>>>>>>> Stashed changes
 class GraphqlQuery(QueryNode):
     def __init__(self):
         QueryNode.__init__(self, 'query')
         self.bind_name = 'data'
 
 
-def create_query(auth, qnode, url, on_error=None):
-    '''
-    Creates a callable object that queries that binds the given 'qnode'
-    with data returned from the graphql query.
-    '''
-    http = requests.Session()
-    http.headers['Authorization'] = 'Bearer {}'.format(auth)
-    root_node = GraphqlQuery()
-    root_node.add_child_node(qnode)
-    query_invcation = functools.partial(query_node,
-                                        root_node,
-                                        http,
-                                        url,
-                                        on_error)
-    qnode.set_query_call(query_invcation)
-    return query_invcation
-
-
 def usage_example():
     query = QueryNode('query')
     linus = query.add_child_node(QueryNode('user')).\
         add_arg('login', 'torvalds')
     linus.add_child_node(QueryNode('id'))
     linus.add_child_node(QueryNode('email'))
-    linus.add_child_node(QueryNode('avatar')).add_arg('size', 20)
+    linus.add_child_node(QueryNode('avatarUrl')).add_arg('size', 20)
     pwd = open('pwd.txt', 'r')
-    query = create_query(pwd.readline().strip(), linus,
-                         'https://api.github.com/graphql')
-    query()
+    http = requests.Session()
+    http.headers['Authorization'] = \
+        'Bearer 7556acd462db83839be375fa29717ba597c56fc0'
```

Comment:
you don't want a private token in source code, not to mention that this is a public repo...